src/renderer_d3d11.cpp

```diff
@@ -2441,7 +2441,7 @@ namespace bgfx { namespace d3d11
 				uint32_t flags = _resolution.reset & (~BGFX_RESET_INTERNAL_FORCE);
 
 				bool resize = true
-					&& !BX_ENABLED(BX_PLATFORM_XBOXONE || BX_PLATFORM_WINRT) // can't use ResizeBuffers on Windows Phone
+					&& !BX_ENABLED(BX_PLATFORM_XBOXONE)
```

Comment:
I don't think anyone cares about running on Windows Phone anymore. If we don't call ResizeBuffers in WinRT, there is a long frame of the background color which results in flashing when resizing a window. Using ResizeBuffers fixes this problem.

Reply:
Yeah, I need to do pass to remove Windows Phone leftovers.